src/ClientFlags/ClientFlags.cpp

```diff
@@ -90,3 +90,5 @@ ABSL_FLAG(bool, time_range_selection, false, "Enable time range selection featur
 
 ABSL_FLAG(bool, tracepoint_callstack_collection, false,
           "Enable callstack gathering for tracepoints feature.");
+
+ABSL_FLAG(bool, symbol_store, false, "Enable symbol store feature.");
```

Comment:
Nit: I would prefer `symbol_store_support`. And as a description text `Enable experimental symbol store support`.